django_lightweight_queue/app_settings.py

```diff
@@ -21,3 +21,7 @@ def setting(suffix, default):
 REDIS_HOST = setting('REDIS_HOST', '127.0.0.1')
 REDIS_PORT = setting('REDIS_PORT', 6379)
 REDIS_PREFIX = setting('REDIS_PREFIX', '')
+
+ENABLE_PROMETHEUS = setting('ENABLE_PROMETHEUS', True)
```

Comment:
Question: if ease of deployment (for us) is our primary concern, why do we default to the synchronous backend rather than one of the Redis based ones? (As a task/queueing system having a synchronous backend isn't very useful on its own, it only makes sense as a default for developers or as a demo.)